tests/integration/test_migrate.py

```diff
@@ -81,8 +84,18 @@ def test_migration_basic(cluster_factory):
             for key in keys:
                 key_names.append(key[0].decode('utf-8'))
 
-            assert cluster1.execute('migrate', '', '', '', '', '', 'keys',
-                                    *key_names) == b'OK'
+            # Try to handle 'migrate' error response gracefully. In the process
+            # of migration, leader node can lose leadership or connection to
+            # other nodes. In this case, we just log the error response and
+            # retry the operation.
```

Comment:
where do we retry the migrate command?

Reply:
We just continue to loop, it solves the issue. Probably, we can do it more explicit, I'll update it.